### ISSUE TITLE
Change the way upsell is handled in the Customizer

### ISSUE THREAD

**Issue by Misplon** [siteorigin/settings#6]
Unfortunately, we're going to need to adapt the way we handle upsell for themes in the Customizer. Namely:
1. One link in the customizer.
2. No options or panels/sections behind a paywall.

I have a feeling that item two above, also refers to the appearance of having options behind a paywall, even if that functionality isn't present in the code of the free theme.

Final clarification will be provided by tonight's TRT meeting.

**Comment by gregpriday** [comment]
@Misplon: what was the final result of this?

**Comment by Misplon** [comment]
@gregpriday meeting summary from JT on this were as follows:

(votes from meeting participants are the numbers at the end of each line item)

**Upselling**

We continued our discussion of what methods for upselling are currently allowed. The following items were discussed and voted upon. Note that some minor changes were made to the proposals from the previous discussion.

Numbers listed are vote counts in favor of adding the guideline. No unfavorable votes were added.

No global nags at top of admin pages. +12
One top-level link in the customizer (other unobtrusive links in sections allowed). +8
Allow one Appearance sub-page. +11
No options or panels/sections behind a paywall. +11
Note that we might change some of the wording for clarity if/when these become guidelines. The admins will review these items for inclusion as guidelines and have further discussion if need be. Overall, there was a majority who voted for these items to be added.

Misplon: So we don't have a final guideline to rely on yet. Current state for this issue is:

_One top-level link in the customizer (other unobtrusive links in sections allowed). +8_
